src/backend/InvenTree/InvenTree/api.py

```diff
@@ -230,14 +230,23 @@ def get(self, request, *args, **kwargs):
             'debug_mode': settings.DEBUG,
             'docker_mode': settings.DOCKER,
             'default_locale': settings.LANGUAGE_CODE,
-            # Following fields are only available to staff users
-            'system_health': check_system_health() if is_staff else None,
-            'database': InvenTree.version.inventreeDatabase() if is_staff else None,
-            'platform': InvenTree.version.inventreePlatform() if is_staff else None,
-            'installer': InvenTree.version.inventreeInstaller() if is_staff else None,
-            'target': InvenTree.version.inventreeTarget() if is_staff else None,
         }
 
+        if is_staff:
```

Comment:
This means that the API response is not uniform and depends on the status of the current user. This makes integrations and probing checks like in https://tools.invenhost.com/check/ more of a hassle; is there a particular reason for this change?